apps/pie-docs/src/components/template/template.md

```diff
@@ -0,0 +1,58 @@
+---
+eleventyNavigation:
+    key: '[Component name]'
+    parent: Components
+    order: 1
+---
+
+## Overview
+[Component Overview]
+
+
+{% contentPageImage {
+    src:""
+} %}
+
+---
+
+## Resources
+
+{% notification {
+  type: "warning",
+  message: "We’re currently working on updating our [Component name] documentation, please see the resources below."
+} %}
+
+{% resourceTable {
+    rows: [
+        {
+            resource: resourceTypes.COMPONENT,
+            link: "google.co.uk",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.DOCUMENTATION,
+            status: statusTypes.PLANNED
+        },
+        {
+            resource: resourceTypes.WEB_COMPONENTS,
+            status: statusTypes.ALPHA
+        },
+        {
+            resource: resourceTypes.VUE,
+            status: statusTypes.TBC
+        },
+        {
+            resource: resourceTypes.REACT,
+            status: statusTypes.DEPRECATED,
+            note: "07/04/1990"
```

Comment:
To avoid ambiguity please can we use the YYYY-MM-DD format?

Reply:
@RLumleyJET

Reply:
Hi both, I have no problem at all if this works better! But just out of curiosity, I am wondering why this format is preferred?

Reply:
Because you never know if dates are dd/mm/yyyy (like in Holland) or mm/dd/yyyy (like in US).

Reply:
Thanks, as it's the draft I'll write it out in that format instead of an actual date so it's cleaere when other people add pages in